.env.example

```diff
@@ -13,3 +13,8 @@ ANALYTICS_MODE=
 # Use `trace` to get verbose logging or `info` to show less
 LOG_LEVEL=debug
 LOGDNA_INGESTION_KEY=
+OPENAI_API_HOST=https://api.openai.com
+OPENAI_API_KEY=
+CHATGPT_SYSTEM_PROMPT_FOR_IMPORTANT_WORDS="You are an 'important words finder'. You need to find important words from given context. You only have to give important words from given context and you have to separate the words by #"
+CHATGPT_USER_PROMPT_FOR_IMPORTANT_WORDS="I need your help to find duplicate issues on my GitHub repository. For context, the entire strategy is the following:\n\n1. A new issue is posted\n2. We ask you to extract a word list of the most \"important\" (i.e. unique adjectives?) words.\n3. We search the repository for all issues with the important words.\n4. We go from highest issue number (most recent) and read the issue description.\n5. If >80% confidence that it's a redundant issue, stop the search and link back to it with a warning saying that it's likely to be a duplicate.\nRight now, we are on step 2.\nPlease separate the words by '#' so I can parse them easily. Please answer simply as I only need the important words."
+SIMILARITY_THRESHOLD=80
```

Comment:
I don't think prompts need to be customizable for this feature.

Reply:
> I don't think prompts need to be customizable for this feature.

So shall I remove from env and leave them just as constants? @pavlovcik

Reply:
> > I don't think prompts need to be customizable for this feature.
> 
> So shall I remove from env and leave them just as constants? @pavlovcik

I changed my mind because I realized that repository maintainers can increase the accuracy of this feature by providing additional context on what their project is about and perhaps even specify specific words to look out for lol

Reply:
Stored configs to .env according to @0xcodercrane

Reply:
No they should be in `ubiquibot-config.json` so that each partner can adjust on their own instance.